aten/src/ATen/native/cuda/SummaryOps.cu

```diff
@@ -289,5 +316,16 @@ Tensor _bincount_cuda(
   });
 }
 
+Tensor _hist_cuda(
+    const Tensor& self,
+    int64_t nbins,
+    scalar_t min,
+    scalar_t max) {
+  Tensor output = native::zeros({nbins}, device(DeviceType::CUDA).dtype(kLong));
+  auto ret = cuda::CUDA_tensor_histogram<int64_t, input_t, false>(
+      output, self, torch.tensor([]), nbins, min, max, 0); // FIXME can weights be optional or null?
```

Comment:
`torch.tensor([])` doesn't work in c++, if you're trying to create an empty tensor you can do it via `at::empty([])` (what are you trying to do here?)

Reply:
yes! I had the doubt too so left myself a FIXME reminder. will fix this